core/unit_test/TestTeam.hpp

```diff
@@ -41,10 +41,9 @@ struct TestTeamPolicy {
     // FIXME_OPENMPTARGET temporary restriction for team size to be at least 32
 #ifdef KOKKOS_ENABLE_OPENMPTARGET
         Kokkos::TeamPolicy<ScheduleType, ExecSpace>(
-            1,
-            std::is_same_v<ExecSpace, Kokkos::Experimental::OpenMPTarget>
-                ? 32
-                : 1)
+            1, std::is_same_v<ExecSpace, Kokkos::Experimental::OpenMPTarget>
+                   ? 32
+                   : 1)
```

Comment:
White-space change?

Reply:
I did not change this file so I assumed this was done for clang-formatting.